Jenkinsfile

```diff
@@ -465,15 +465,13 @@ pipeline {
                     }
                     post {
                         always {
-                            node('lightweight') {
-                                recordIssues enabledForFailure: true,
-                                             aggregatingResults: true,
-                                             id: "analysis-gcc-centos7",
-                                             tools: [ gcc4(pattern: 'centos7-gcc-build.log'),
-                                                      cppCheck(pattern: 'centos7-gcc-build.log') ],
-                                             filters: [excludeFile('.*\\/_build\\.external\\/.*'),
-                                                       excludeFile('_build\\.external\\/.*')]
-                            }
+                            recordIssues enabledForFailure: true,
```

Comment:
This is mostly removing the node('lightweight') section is it?  What effect does that have?

Reply:
It should be a NOOP as it should fall back to the default agent for the pipeline: https://github.com/daos-stack/daos/blob/f0bc27fd82f876af89a326875d13ada0d639af97/Jenkinsfile#L185

Reply:
Not a noop.  The agent stage puts the post processing in its own workspace, so it can not see any of the artifacts from the stage.